ESP32&mega-20220809 AdaGFX helper issue

Hi all!
Just checked latest mega-20220809 with Wemos D32 mini pro - it's look like an error in AdaGFX helper was expected.
Mega-20220809 was uploaded to clean Wemos D32 mini pro, it can work without issues due ILI9341 plugin was enabled.
After it board stopping to boot at ADAGFX helper initialization step:
```
INIT : Booting version: mega-20220809_57b0c0c (ESP32 SDK v4.4.3)
48 : Info   : INIT : Free RAM:267076
50 : Info   : INIT : Cold Boot - Restart Reason: CPU0: RTC Watch dog reset digital core and rtc module CPU1: for APP CPU, reseted by PRO CPU
51 : Info   : FS   : Mounting...
92 : Info   : FS   : Mount successful, used 151552 bytes of 8318976
175 : Info   : CRC  : SecuritySettings CRC   ...OK
198 : Info   : INIT : I2C
201 : Info   : INIT : SPI Init (without CS)
203 : Info   : ExtRTC: Read external time source: [1660235047](tel:1660235047)
204 : Info   : Time set to [1660235047](tel:1660235047).000
206 : Info   : Current Time Zone: STD time start: 2022-10-30 03:00:00 offset: 360 min
207 : Info   : Local time: 2022-08-11 22:24:07
209 : Info   : WiFi : Start network scan all channels
311 : Info   : WiFi : STA Started
6927 : Info   : WiFi : Scan finished, found: 10
6931 : Info   : WiFi : Best AP candidate: XXXXXXXXXXXXXXXXXXXX Ch:9 (-58dBm) WPA2/PSK
6933 : Info   : WIFI : Connecting XXXXXXXXXXXXXXXXXXXXX Ch:9 (-58dBm) WPA2/PSK attempt #0
6943 : Info   : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
6967 : Info   : INIT : Free RAM:219656
7014 : Info   : ILI9341: Init start.
7015 : Info   : ILI9341: Init done, address: 0x3ffd3abc valid, commands: tft/tftcmd
7016 : Info   : AdaGFX_helper:
```
Web server wasn't yet enabled, therefore only full reset can help to boot board again (with disabled ILI9341). I'll try to check latest build with @tonhuisman  commit #3903

So, same bootloop with new board, wasn't previously used yet. Was flashed with mega-20220809 and minimally configured - same Advanced settings, same Hardware and VSPI. GPIO2&32 was manually changed to 1 by "Commands" through web server. After Display was added it send Core panic, rebooted and stucked at AdaGFX helper step:

> ets Jul 29 2019 12:21:46
> 
> rst:0x1 (POWERON_RESET),boot:0x17 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U42 : Info   :
> 
> INIT : Booting version: mega-20220809_57b0c0c (ESP32 SDK v4.4.3)
> 43 : Info   : INIT : Free RAM:267396
> 45 : Info   : INIT : Cold Boot - Restart Reason: CP./components/esp_littlefs/src                                                            /littlefs/lfs.c:1229:error: Corrupted dir pair at {0x0, 0x1}
> U0: Vbat power on reset CPU1: for APP CPU, reseted by PRO CPU
> 46 : Info   : FS   : Mounting...
> FS   : Mount failed
> 54 : Error  : FS   : Mount failed
> RESET./components/esp_littlefs/src/littlefs/lfs.c:1229:error: Corrupted dir pair                                                             at {0x0, 0x1}
> : Resetting factory defaults... using default settings
> RESET: Warm boot, reset count: 0
> RESET: formatting...
> RESET: formatting done...
> 31170 : Info   : SaveToFile: free stack: 7416
> 33202 : Info   : FILE : Saved config.dat offset: 8192 size: 820
> 33203 : Info   : SaveToFile: free stack after: 7416
> 33204 : Info   : SaveToFile: free stack: 7448
> 35374 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 35375 : Info   : SaveToFile: free stack after: 7448
> 35377 : Info   : SaveToFile: free stack: 7256
> 35488 : Info   : FILE : Saved security.dat offset: 0 size: 593
> 35489 : Info   : SaveToFile: free stack after: 7256
> 35501 : Info   : SaveToFile: free stack: 7000
> 35587 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 35588 : Info   : SaveToFile: free stack after: 7000
> RESET: Successful, rebooting. (you might need to press the reset button if you've just flashed the firmware)
> 36771 : Info   : WIFI : Set WiFi to OFFets Jul 29 2019 12:21:46
> 
> rst:0xc (SW_CPU_RESET),boot:0x17 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U47 : Info   :
> 
> INIT : Booting version: mega-20220809_57b0c0c (ESP32 SDK v4.4.3)
> 48 : Info   : INIT : Free RAM:267076
> 50 : Info   : INIT : Soft Reboot #1 Last Action before Reboot: Intended Reboot: ResetFactory Last systime: 36 - Restart Reason: CPU0: Software reset CPU CPU1: Software reset CPU
> 51 : Info   : FS   : Mounting...
> 89 : Info   : FS   : Mount successful, used 151552 bytes of 8318976
> 175 : Info   : CRC  : SecuritySettings CRC   ...OK
> 197 : Info   : INIT : I2C
> 199 : Info   : INIT : SPI not enabled
> 200 : Info   : Time set to 36.000 Time adjusted by 35800.00 msec. Wander: 0.000 msec/second Source: RTC at boot
> 202 : Info   : Current Time Zone: STD time start: 1970-10-25 03:00:00 offset: 0 min
> 203 : Info   : Local time: 1970-01-01 00:00:36
> 205 : Info   : WiFi : Start network scan all channels
> 305 : Info   : WiFi : STA Started
> 6720 : Info   : WiFi : Scan finished, found: 9
> 6723 : Error  : WIFI : No valid wifi settings
> 6825 : Info   : WIFI : Set WiFi to AP+STA
> 6828 : Info   : WiFi : AP Started
> 7390 : Info   : WiFi : AP Stopped
> 7391 : Info   : WiFi : AP Started
> 7392 : Info   : WIFI : AP Mode ssid will be MAX-ESP32 with address 192.168.4.1
> 7394 : Info   : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 7500 : Info   : WIFI : Disconnected! Reason: '(8) Assoc leave' Connected for 7398 ms
> 7515 : Error  : WiFi : Scan not allowed, unprocessed WiFi events:  disconn
> 7617 : Info   : WIFI : Set WiFi to OFF
> 7842 : Info   : WiFi : Start network scan all channels
> 7854 : Info   : WiFi : STA Started
> 14468 : Info   : WiFi : Scan finished, found: 12
> 14575 : Info   : WIFI : Set WiFi to AP+STA
> 15139 : Info   : WIFI : AP Mode ssid will be MAX-ESP32 with address 192.168.4.1
> 15241 : Info   : WIFI : Disconnected! Reason: '(8) Assoc leave' Connected for 14 s
> 15257 : Error  : WiFi : Scan not allowed, unprocessed WiFi events:  disconn
> 15358 : Info   : WIFI : Set WiFi to OFF
> 15583 : Info   : WIFI : Arduino wifi status: WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 15686 : Info   : WIFI : Set WiFi to AP
> 15697 : Info   : WiFi : AP Started
> 16258 : Info   : WiFi : AP Stopped
> 16259 : Info   : WiFi : AP Started
> 16260 : Info   : WIFI : AP Mode ssid will be MAX-ESP32 with address 192.168.4.1
> 16296 : Info   : INIT : Free RAM:214892
> 16877 : Info   : INFO : Plugins: 124 [Normal] [MAX ESP32] (ESP32 SDK v4.4.3)
> 16882 : Info   : WIFI : Arduino wifi status: WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 16887 : Info   : Webserver: start
> 16903 : Info   : OTA  : Arduino OTA enabled on port 3232
> 18256 : Info   : WD   : Uptime 0 ConnectFailures 0 FreeMem 195232 WiFiStatus WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 26477 : Info   : AP Mode: Client connected: 54:27:1E:06:27:B4 Connected devices: 1
> 26482 : Error  : UNKNOWN WIFI/ETH EVENT: 14
> 48244 : Info   : WD   : Uptime 1 ConnectFailures 0 FreeMem 189848 WiFiStatus WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 50378 : Info   : SaveToFile: free stack: 7208
> 52563 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 52564 : Info   : SaveToFile: free stack after: 7208
> 52565 : Info   : SaveToFile: free stack: 7016
> 52671 : Info   : FILE : Saved security.dat offset: 0 size: 593
> 52672 : Info   : SaveToFile: free stack after: 7016
> 52689 : Info   : SaveToFile: free stack: 6760
> 52780 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 52781 : Info   : SaveToFile: free stack after: 6760
> 52783 : Info   : WIFI : Credentials Changed, retry connection. SSID: access_point8
> 52785 : Info   : WiFi : Best AP candidate: access_point8 88:DC:96:26:7F:0A Ch:9 (-61dBm) WPA2/PSK
> 52786 : Info   : WIFI : Connecting access_point8 88:DC:96:26:7F:0A Ch:9 (-61dBm) WPA2/PSK attempt #0
> 52887 : Info   : WIFI : Set WiFi to AP+STA
> 52891 : Info   : WiFi : STA Started
> 52997 : Info   : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 53895 : Info   : WIFI : Connected! AP: access_point8 (88:DC:96:26:7F:0A) Ch: 9 Duration: 1102 ms
> 53912 : Info   : WIFI : DHCP IP: 192.168.3.25 (MAX-ESP32) GW: 192.168.3.1 SN: 255.255.255.0   duration: 20 ms
> 53930 : Info   : SaveToFile: free stack: 7096
> 54019 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 54021 : Info   : SaveToFile: free stack after: 7096
> 54022 : Info   : WIFI : Arduino wifi status: WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP
> 54022 : Info   : WIFI : Starting mDNS...
> 54026 : Info   : WIFI : mDNS started, with name: MAX-ESP32.local
> 54040 : Info   : UDP : Start listening on port 8266
> 54041 : Info   : firstLoopConnectionsEstablished
> 55666 : Info   : AP Mode: Client disconnected: 54:27:1E:06:27:B4 Connected devices: 1
> 60503 : Info   : AP Mode: Client connected: 54:27:1E:06:27:B4 Connected devices: 1
> 60514 : Error  : UNKNOWN WIFI/ETH EVENT: 14
> 78237 : Info   : WD   : Uptime 1 ConnectFailures 0 FreeMem 183432 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 99519 : Info   :  Webserver 33 Arguments (First 20): 0: 'userules' length: 2 1: 'enable_rules_cache' length: 2 2: 'usentp' length: 2 3: 'ntphost' length: 11 4: 'exttimesource' length: 1 5: 'dststartweek' length: 1 6: 'dststartdow' length: 1 7: 'dststartmonth' length: 1 8: 'dststarthour' length: 1 9: 'dstendweek' length: 1 10: 'dstenddow' length: 1 11: 'dstendmonth' length: 2 12: 'dstendhour' length: 1 13: 'timezone' length: 3 14: 'latitude' length: 8 15: 'longitude' length: 8 16: 'syslogip' length: 0 17: 'syslogport' length: 3 18: 'syslog_log_level' length: 1 19: 'syslogfacility' length: 1
> 99527 : Info   : SaveToFile: free stack: 7240
> 101708 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 101709 : Info   : SaveToFile: free stack after: 7240
> 101727 : Info   : SaveToFile: free stack: 6792
> 101824 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 101826 : Info   : SaveToFile: free stack after: 6792
> 101842 : Info   : NTP  : NTP replied: delay 10 mSec Accuracy increased by 0.007 seconds
> 101845 : Error  : ExtRTC: Cannot set time to external time source
> 101845 : Info   : Time set to 1660364288.007
> 101846 : Info   : Current Time Zone: STD time start: 2022-10-30 03:00:00 offset: 360 min
> 101848 : Info   : Local time: 2022-08-13 10:18:08
> 102007 : Info   : EVENT: Time#Set
> 102231 : Info   : EVENT: Clock#Time=Sat,10:18
> 108238 : Info   : WD   : Uptime 2 ConnectFailures 0 FreeMem 179404 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 133899 : Info   : Upload: START, filename: rules1.txt
> 133917 : Info   : Upload: WRITE, Bytes: 85
> 133922 : Info   : Upload: END, Size: 85
> 134004 : Info   :  Webserver 1 Arguments: 0: 'enctype' length: 19
> 134048 : Info   : CRC  : SecuritySettings CRC   ...OK
> 138237 : Info   : WD   : Uptime 2 ConnectFailures 0 FreeMem 181100 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 154231 : Info   : EVENT: Clock#Time=Sat,10:19
> 168237 : Info   : WD   : Uptime 3 ConnectFailures 0 FreeMem 181792 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 170507 : Info   :  Webserver 41 Arguments (First 20): 0: 'pled' length: 2 1: 'pledi' length: 2 2: 'pres' length: 2 3: 'psda' length: 2 4: 'pscl' length: 2 5: 'pi2csp' length: 6 6: 'pi2cspslow' length: 6 7: 'pi2cmuxtype' length: 2 8: 'pi2cmuxaddr' length: 2 9: 'pi2cmuxreset' length: 2 10: 'initspi' length: 1 11: 'spipinsclk' length: 2 12: 'spipinmiso' length: 2 13: 'spipinmosi' length: 2 14: 'sd' length: 2 15: 'p0' length: 1 16: 'p2' length: 1 17: 'p4' length: 1 18: 'p5' length: 1 19: 'p12' length: 1
> 170513 : Info   : SaveToFile: free stack: 7048
> 172669 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 172670 : Info   : SaveToFile: free stack after: 7048
> 172684 : Info   : SaveToFile: free stack: 6600
> 172773 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 172775 : Info   : SaveToFile: free stack after: 6600
> 172776 : Info   : INIT : I2C
> 197206 : Info   :  Webserver 1 Arguments: 0: 'cmd' length: 9
> 197207 : Info   : HTTP: GPIO,32,1
> 197214 : Info   : GPIO : port#32: set to 1
> 198237 : Info   : WD   : Uptime 3 ConnectFailures 0 FreeMem 181216 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 202844 : Info   :  Webserver 1 Arguments: 0: 'cmd' length: 8
> 202845 : Info   : HTTP: GPIO,2,1
> 202851 : Info   : GPIO : port#2: set to 1
> 214231 : Info   : EVENT: Clock#Time=Sat,10:20
> 217490 : Info   :  Webserver 2 Arguments: 0: 'index' length: 1 1: 'page' length: 1
> 224655 : Info   :  Webserver 5 Arguments: 0: 'index' length: 1 1: 'page' length: 1 2: 'TDNUM' length: 2 3: 'edit' length: 1 4: 'page' length: 1
> 226274 : Info   : SaveToFile: free stack: 7096
> 227882 : Info   : FILE : Saved config.dat offset: 32768 size: 536
> 227883 : Info   : SaveToFile: free stack after: 7096
> 227885 : Info   : SaveToFile: free stack: 7176
> 230023 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 230024 : Info   : SaveToFile: free stack after: 7176
> 230041 : Info   : SaveToFile: free stack: 6728
> 230135 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 230136 : Info   : SaveToFile: free stack after: 6728
> 230451 : Info   : EVENT: TaskExit#=1,0
> 230542 : Info   : WD   : Uptime 4 ConnectFailures 0 FreeMem 180668 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 258239 : Info   : WD   : Uptime 4 ConnectFailures 0 FreeMem 181792 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 258986 : Info   :  Webserver 49 Arguments (First 20): 0: 'index' length: 1 1: 'page' length: 1 2: 'TDNUM' length: 2 3: 'TDN' length: 7 4: 'TDE' length: 2 5: 'taskdevicepin1' length: 2 6: 'taskdevicepin2' length: 2 7: 'taskdevicepin3' length: 2 8: 'p095_backlight' length: 2 9: 'p095_backpercentage' length: 3 10: 'p095_button' length: 2 11: 'p095_timer' length: 1 12: 'p095_rotate' length: 1 13: 'p095_mode' length: 1 14: 'p095_fontscale' length: 1 15: 'p095_commandtrigger' length: 1 16: 'p095_compat' length: 2 17: 'p095_backfill' length: 2 18: 'p095_foregroundcolor' length: 5 19: 'p095_backgroundcolor' length: 5
> 259010 : Info   : SaveToFile: free stack: 6792
> 260634 : Info   : FILE : Saved config.dat offset: 32768 size: 536
> 260635 : Info   : SaveToFile: free stack after: 6792
> 260700 : Info   : SaveToFile: free stack: 5912
> 262306 : Info   : FILE : Saved config.dat offset: 33792 size: 24
> 262307 : Info   : SaveToFile: free stack after: 5912
> 262325 : Info   : SaveToFile: free stack: 7096
> 263924 : Info   : FILE : Saved config.dat offset: 32768 size: 536
> 263925 : Info   : SaveToFile: free stack after: 7096
> 263927 : Info   : SaveToFile: free stack: 7176
> 266087 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 266088 : Info   : SaveToFile: free stack after: 7176
> 266112 : Info   : SaveToFile: free stack: 6728
> 266215 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 266217 : Info   : SaveToFile: free stack after: 6728
> 266268 : Info   : ILI9341: Init start.
> 266269 : Info   : ILI9341: Init done, address: 0x3ffd10dc valid, commands: tft/tftcmd
> 266270 : Info   : AdaGFXGuru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
> 
> Core  1 register dump:
> PC      : 0x401e53fd  PS      : 0x00060730  A0      : 0x800d2bc4  A1      : 0x3ffb2140
> A2      : 0x00000000  A3      : 0x3ffc2c04  A4      : 0x00000040  A5      : 0x00000000
> A6      : 0x00ff0000  A7      : 0xff000000  A8      : 0x801e3fdc  A9      : 0x3ffb2120
> A10     : 0x00000000  A11     : 0x00000001  A12     : 0x00000010  A13     : 0x3ffc6a68
> A14     : 0x40263738  A15     : 0x3ffdab28  SAR     : 0x00000005  EXCCAUSE: 0x0000001c
> EXCVADDR: 0x00000000  LBEG    : 0x400d38ac  LEND    : 0x400d38b0  LCOUNT  : 0x00000000
> 
> 
> Backtrace:0x401e53fa:0x3ffb21400x400d2bc1:0x3ffb2170 0x400d383d:0x3ffb2190 0x400d38c2:0x3ffb21b0 0x400d3955:0x3ffb21d0 0x400d3a93:0x3ffb21f0 0x4026374f:0x3ffb2210 0x401b9afe:0x3ffb2230 0x401453e7:0x3ffb22a0 0x4018c36c:0x3ffb24c0 0x401de0f3:0x3ffb25b0 0x4026e4e5:0x3ffb2660 0x400d7f8a:0x3ffb2680 0x400dcb61:0x3ffb26a0 0x400dcc11:0x3ffb26e0 0x400dcd19:0x3ffb2740 0x4018964e:0x3ffb2790 0x401898fd:0x3ffb27c0 0x40123f87:0x3ffb2800 0x401e5eb8:0x3ffb2820
> 
> 
> 
> 
> ELF file SHA256: 0000000000000000
> 
> Rebooting...
> ets Jul 29 2019 12:21:46
> 
> rst:0xc (SW_CPU_RESET),boot:0x17 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U47 : Info   :
> 
> INIT : Booting version: mega-20220809_57b0c0c (ESP32 SDK v4.4.3)
> 48 : Info   : INIT : Free RAM:267076
> 50 : Info   : INIT : Soft Reboot #2 Last Action before Reboot: Background Task Last systime: 1660364444 - Restart Reason: CPU0: Software reset CPU CPU1: Software reset CPU
> 51 : Info   : FS   : Mounting...
> 114 : Info   : FS   : Mount successful, used 151552 bytes of 8318976
> 222 : Info   : CRC  : SecuritySettings CRC   ...OK
> 251 : Info   : INIT : I2C
> 254 : Info   : INIT : SPI Init (without CS)
> 256 : Info   : ExtRTC: Read external time source: 1660364449
> 257 : Info   : Time set to 1660364449.000
> 258 : Info   : Current Time Zone: STD time start: 2022-10-30 03:00:00 offset: 360 min
> 260 : Info   : Local time: 2022-08-13 10:20:49
> 262 : Info   : WiFi : Start network scan all channels
> 363 : Info   : WiFi : STA Started
> 6980 : Info   : WiFi : Scan finished, found: 13
> 6983 : Info   : WiFi : Best AP candidate: access_point8 88:DC:96:26:7F:0A Ch:9 (-56dBm) WPA2/PSK
> 6985 : Info   : WIFI : Connecting access_point8 88:DC:96:26:7F:0A Ch:9 (-56dBm) WPA2/PSK attempt #0
> 6995 : Info   : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 7019 : Info   : INIT : Free RAM:219540
> 7072 : Info   : ILI9341: Init start.
> 7073 : Info   : ILI9341: Init done, address: 0x3ffd3ab4 valid, commands: tft/tftcmd
> 7074 : Info   : AdaGFX_helper

Test scheme is:
![image](https://user-images.githubusercontent.com/55306010/184468408-90c96f55-c362-4256-a1bc-0462ad05b659.png)

> Can you please test this fresh build (flash the factory.bin file via Serial please to have the latest bootloader) to see if that sticks better on your board?

No result, same Core panic after PMS7003 activation:

> ets Jun  8 2016 00:22:57
> 
> rst:0x1 (POWERON_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U42 : Info   :
> 
> INIT : Booting version: ESP_Easy_mega_20220813_max_ESP32_16M8M_LittleFS, (Self built) feature/several-small-improvements-2022-08_5bd455d (ESP32 SDK v4.4../components/esp_littlefs/src/littlefs/lfs.c:1071:error: Corrupted dir pair at {0x0, 0x1}
> 3)
> 44 : Info   : INIT : Free RAM:266928
> 45 : Info   : INIT : Cold Boot - Restart Reason: CPU0: Vbat power on reset CPU1: for APP CPU, reseted by PRO CPU
> 46./components/esp_littlefs/src/littlefs/lfs.c:1071:error: Corrupted dir pair at {0x0, 0x1}
>  : Info   : FS   : Mounting...
> FS   : Mount failed
> 54 : Error  : FS   : Mount failed
> RESET: Resetting factory defaults... using default settings
> RESET: Warm boot, reset count: 0
> RESET: formatting...
> RESET: formatting done...
> 8015 : Info   : SaveToFile: free stack: 7416
> 9483 : Info   : FILE : Saved config.dat offset: 8192 size: 820
> 9484 : Info   : SaveToFile: free stack after: 7416
> 9486 : Info   : SaveToFile: free stack: 7448
> 11060 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 11061 : Info   : SaveToFile: free stack after: 7448
> 11063 : Info   : SaveToFile: free stack: 7256
> 11156 : Info   : FILE : Saved security.dat offset: 0 size: 593
> 11157 : Info   : SaveToFile: free stack after: 7256
> 11169 : Info   : SaveToFile: free stack: 7000
> 11237 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 11238 : Info   : SaveToFile: free stack after: 7000
> RESET: Successful, rebooting. (you might need to press the reset button if you've just flashed the firmware)
> 12419 : Info   : WIFI : Set WiFi to OFFets Jun  8 2016 00:22:57
> 
> rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U46 : Info   :
> 
> INIT : Booting version: ESP_Easy_mega_20220813_max_ESP32_16M8M_LittleFS, (Self built) feature/several-small-improvements-2022-08_5bd455d (ESP32 SDK v4.4.3)
> 48 : Info   : INIT : Free RAM:266608
> 49 : Info   : INIT : Soft Reboot #1 Last Action before Reboot: Intended Reboot: ResetFactory Last systime: 12 - Restart Reason: CPU0: Software reset CPU CPU1: Software reset CPU
> 50 : Info   : FS   : Mounting...
> 88 : Info   : FS   : Mount successful, used 151552 bytes of 8318976
> 174 : Info   : CRC  : SecuritySettings CRC   ...OK
> 195 : Info   : INIT : I2C
> 198 : Info   : INIT : SPI not enabled
> 199 : Info   : Time set to 12.000 Time adjusted by 11801.00 msec. Wander: 0.000 msec/second Source: RTC at boot
> 201 : Info   : Current Time Zone: STD time start: 1970-10-25 03:00:00 offset: 0 min
> 202 : Info   : Local time: 1970-01-01 00:00:12
> 204 : Info   : WiFi : Start network scan all channels
> 306 : Info   : WiFi : STA Started
> 6921 : Info   : WiFi : Scan finished, found: 11
> 6925 : Error  : WIFI : No valid wifi settings
> 7026 : Info   : WIFI : Set WiFi to AP+STA
> 7030 : Info   : WiFi : AP Started
> 7591 : Info   : WIFI : AP Mode ssid will be MAX-ESP32 with address 192.168.4.1
> 7594 : 7595 : Info   : WiFi : AP Stopped
> Info   7595 : Info   : WiFi : AP Started
>                                          : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 7703 : Info   : WIFI : Disconnected! Reason: '(8) Assoc leave' Connected for 7601 ms
> 7718 : Error  : WiFi : Scan not allowed, unprocessed WiFi events:  disconn
> 7819 : Info   : WIFI : Set WiFi to OFF
> 8044 : Info   : WiFi : Start network scan all channels
> 8054 : Info   : WiFi : STA Started
> 14669 : Info   : WiFi : Scan finished, found: 10
> 14776 : Info   : WIFI : Set WiFi to AP+STA
> 15340 : Info   : WIFI : AP Mode ssid will be MAX-ESP32 with address 192.168.4.1
> 15442 : Info   : WIFI : Disconnected! Reason: '(8) Assoc leave' Connected for 14 s
> 15458 : Error  : WiFi : Scan not allowed, unprocessed WiFi events:  disconn
> 15559 : Info   : WIFI : Set WiFi to OFF
> 15784 : Info   : WIFI : Arduino wifi status: WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 15887 : Info   : WIFI : Set WiFi to AP
> 15898 : Info   : WiFi : AP Started
> 16459 : Info   : WiFi : AP Stopped
> 16460 : Info   : WiFi : AP Started
> 16460 : Info   : WIFI : AP Mode ssid will be MAX-ESP32 with address 192.168.4.1
> 16497 : Info   : INIT : Free RAM:214608
> 17077 : Info   : INFO : Plugins: 126 [Normal] [MAX ESP32] (ESP32 SDK v4.4.3)
> 17081 : Info   : WIFI : Arduino wifi status: WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 17086 : Info   : Webserver: start
> 17092 : Info   : OTA  : Arduino OTA enabled on port 3232
> 18435 : Info   : WD   : Uptime 0 ConnectFailures 0 FreeMem 194760 WiFiStatus WL_NO_SHIELD 255 ESPeasy internal wifi status: DISCONNECTED
> 18909 : Info   : AP Mode: Client connected: 54:27:1E:06:27:B4 Connected devices: 1
> 18925 : Error  : UNKNOWN WIFI/ETH EVENT: 14
> 46232 : Info   : SaveToFile: free stack: 7208
> 47799 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 47801 : Info   : SaveToFile: free stack after: 7208
> 47802 : Info   : SaveToFile: free stack: 7016
> 47891 : Info   : FILE : Saved security.dat offset: 0 size: 593
> 47892 : Info   : SaveToFile: free stack after: 7016
> 47908 : Info   : SaveToFile: free stack: 6760
> 47981 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 47982 : Info   : SaveToFile: free stack after: 6760
> 47983 : Info   : WIFI : Credentials Changed, retry connection. SSID: access_point8
> 47986 : Info   : WiFi : Best AP candidate: access_point8 88:DC:96:26:7F:0A Ch:9 (-59dBm) WPA2/PSK
> 47987 : Info   : WIFI : Connecting access_point8 88:DC:96:26:7F:0A Ch:9 (-59dBm) WPA2/PSK attempt #0
> 48089 : Info   : WIFI : Set WiFi to AP+STA
> 48092 : Info   : WiFi : STA Started
> 48199 : Info   : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 48464 : Info   : WD   : Uptime 1 ConnectFailures 0 FreeMem 186276 WiFiStatus WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 49125 : Info   : WIFI : Connected! AP: access_point8 (88:DC:96:26:7F:0A) Ch: 9 Duration: 1132 ms
> 49145 : Info   : WIFI : DHCP IP: 192.168.3.3 (MAX-ESP32) GW: 192.168.3.1 SN: 255.255.255.0   duration: 24 ms
> 49164 : Info   : SaveToFile: free stack: 7096
> 49238 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 49239 : Info   : SaveToFile: free stack after: 7096
> 49240 : Info   : WIFI : Arduino wifi status: WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP
> 49241 : Info   : WIFI : Starting mDNS...
> 49245 : Info   : WIFI : mDNS started, with name: MAX-ESP32.local
> 49258 : Info   : UDP : Start listening on port 8266
> 49260 : Info   : firstLoopConnectionsEstablished
> 50950 : Info   : AP Mode: Client disconnected: 54:27:1E:06:27:B4 Connected devices: 1
> 55783 : Info   : AP Mode: Client connected: 54:27:1E:06:27:B4 Connected devices: 1
> 55788 : Error  : UNKNOWN WIFI/ETH EVENT: 14
> 78427 : Info   : WD   : Uptime 1 ConnectFailures 0 FreeMem 181916 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 93086 : Info   :  Webserver 33 Arguments (First 20): 0: 'userules' length: 2 1: 'enable_rules_cache' length: 2 2: 'usentp' length: 2 3: 'ntphost' length: 11 4: 'exttimesource' length: 1 5: 'dststartweek' length: 1 6: 'dststartdow' length: 1 7: 'dststartmonth' length: 1 8: 'dststarthour' length: 1 9: 'dstendweek' length: 1 10: 'dstenddow' length: 1 11: 'dstendmonth' length: 2 12: 'dstendhour' length: 1 13: 'timezone' length: 3 14: 'latitude' length: 8 15: 'longitude' length: 8 16: 'syslogip' length: 0 17: 'syslogport' length: 3 18: 'syslog_log_level' length: 1 19: 'syslogfacility' length: 1
> 93093 : Info   : SaveToFile: free stack: 7240
> 94662 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 94664 : Info   : SaveToFile: free stack after: 7240
> 94682 : Info   : SaveToFile: free stack: 6792
> 94760 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 94762 : Info   : SaveToFile: free stack after: 6792
> 94778 : Info   : NTP  : NTP replied: delay 11 mSec Accuracy increased by 0.674 seconds
> 94781 : Error  : ExtRTC: Cannot set time to external time source
> 94781 : Info   : Time set to 1660387175.674
> 94782 : Info   : Current Time Zone: STD time start: 2022-10-30 03:00:00 offset: 360 min
> 94784 : Info   : Local time: 2022-08-13 16:39:35
> 94949 : Info   : EVENT: Time#Set
> 95165 : Info   : EVENT: Clock#Time=Sat,16:39
> 108427 : Info   : WD   : Uptime 2 ConnectFailures 0 FreeMem 181756 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 118596 : Info   :  Webserver 41 Arguments (First 20): 0: 'pled' length: 2 1: 'pledi' length: 2 2: 'pres' length: 2 3: 'psda' length: 2 4: 'pscl' length: 2 5: 'pi2csp' length: 6 6: 'pi2cspslow' length: 6 7: 'pi2cmuxtype' length: 2 8: 'pi2cmuxaddr' length: 2 9: 'pi2cmuxreset' length: 2 10: 'initspi' length: 1 11: 'spipinsclk' length: 2 12: 'spipinmiso' length: 2 13: 'spipinmosi' length: 2 14: 'sd' length: 2 15: 'p0' length: 1 16: 'p2' length: 1 17: 'p4' length: 1 18: 'p5' length: 1 19: 'p12' length: 1
> 118602 : Info   : SaveToFile: free stack: 7048
> 120131 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 120133 : Info   : SaveToFile: free stack after: 7048
> 120139 : Info   : SaveToFile: free stack: 6600
> 120199 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 120201 : Info   : SaveToFile: free stack after: 6600
> 120201 : Info   : INIT : I2C
> 120467 : Info   : EVENT: Clock#Time=Sat,16:40
> 136722 : Info   :  Webserver 33 Arguments (First 20): 0: 'userules' length: 2 1: 'enable_rules_cache' length: 2 2: 'usentp' length: 2 3: 'ntphost' length: 11 4: 'exttimesource' length: 1 5: 'dststartweek' length: 1 6: 'dststartdow' length: 1 7: 'dststartmonth' length: 1 8: 'dststarthour' length: 1 9: 'dstendweek' length: 1 10: 'dstenddow' length: 1 11: 'dstendmonth' length: 2 12: 'dstendhour' length: 1 13: 'timezone' length: 3 14: 'latitude' length: 8 15: 'longitude' length: 8 16: 'syslogip' length: 0 17: 'syslogport' length: 3 18: 'syslog_log_level' length: 1 19: 'syslogfacility' length: 1
> 136729 : Info   : SaveToFile: free stack: 7240
> 138269 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 138272 : Info   : SaveToFile: free stack after: 7240
> 138273 : Debug dev : Write settings: ExtendedControllerCredentials index: 0
> 138284 : Info   : SaveToFile: free stack: 6792
> 138350 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 138352 : Info   : SaveToFile: free stack after: 6792
> 138354 : Info   : ExtRTC: Read external time source: 1660387218
> 138355 : Info   : Time set to 1660387218.000 Time adjusted by -1249.28 msec. Wander: -28.670 msec/second Source: Ext. RTC at boot
> 138357 : Info   : Local time: 2022-08-13 16:40:18
> 138363 : Debug dev : sendcontent free: 177576 chunk size:1360
> 138380 : Debug dev : sendcontent free: 176128 chunk size:1360
> 138398 : Debug dev : sendcontent free: 177528 chunk size:1360
> 138411 : Debug dev : sendcontent free: 175796 chunk size:1360
> 138428 : Debug dev : sendcontent free: 178004 chunk size:1360
> 138442 : Debug dev : sendcontent free: 176468 chunk size:1360
> 138453 : Debug dev : sendcontent free: 174616 chunk size:1360
> 138466 : Debug dev : sendcontent free: 176316 chunk size:1360
> 138483 : Debug dev : sendcontent free: 177956 chunk size:1360
> 138500 : Debug dev : sendcontent free: 176424 chunk size:1360
> 138511 : Debug dev : sendcontent free: 169824 chunk size:291
> 138520 : Debug dev : sendcontent free: 172916 chunk size:0
> 138541 : Info   : EVENT: Time#Set
> 138569 : Debug  : Rules : Read 0 lines from rules1.txt
> 138585 : Debug  : EVENT: Time#Set Processing time:44 milliSeconds
> 138598 : Info   : WD   : Uptime 2 ConnectFailures 0 FreeMem 180024 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 138599 : Debug More : UDP  : Send Sysinfo message
> 139316 : Debug  : LoopStats: shortestLoop: 47 longestLoop: 2088719 avgLoopDuration: 77.71 loopCounterMax: 638297 loopCounterLast: 378579
> 139317 : Debug  : Scheduler stats: (called/tasks/max_length/idle%) 1201163/7662/6/87.43
> 144454 : Debug dev : sendcontent free: 178136 chunk size:1360
> 144466 : Debug dev : sendcontent free: 176348 chunk size:1360
> 144478 : Debug dev : sendcontent free: 178136 chunk size:1360
> 144485 : Debug dev : sendcontent free: 171408 chunk size:1360
> 144498 : Debug dev : sendcontent free: 179672 chunk size:1360
> 144514 : Debug dev : sendcontent free: 176352 chunk size:1360
> 144526 : Debug dev : sendcontent free: 172936 chunk size:1360
> 144534 : Debug dev : sendcontent free: 166204 chunk size:1360
> 144551 : Debug dev : sendcontent free: 174780 chunk size:92
> 144561 : Debug dev : sendcontent free: 176300 chunk size:0
> 147612 : Debug dev : sendcontent free: 177904 chunk size:1360
> 147625 : Debug dev : sendcontent free: 179496 chunk size:1360
> 147637 : Debug dev : sendcontent free: 177912 chunk size:1360
> 147648 : Debug dev : sendcontent free: 176180 chunk size:1360
> 147661 : Debug dev : sendcontent free: 177912 chunk size:1360
> 147673 : Info   :  Webserver 1 Arguments: 0: 'cmd' length: 9
> 147675 : Info   : HTTP: GPIO,32,1
> 147677 : Debug  : HTTP after parseTemplate: GPIO,32,1
> 147680 : Debug  : Command: GPIO
> 147681 : Debug  : GPIO,32,1
> 147681 : Debug  : Par1: 32 Par2: 1 Par3: 0 Par4: 0 Par5: 0
> 147683 : Info   : GPIO : port#32: set to 1
> 147688 : Debug dev : sendcontent free: 179336 chunk size:1360
> 147702 : Debug dev : sendcontent free: 172660 chunk size:1360
> 147712 : Debug dev : sendcontent free: 175892 chunk size:1360
> 147723 : Debug dev : sendcontent free: 177548 chunk size:274
> 147733 : Debug dev : sendcontent free: 175956 chunk size:0
> 153058 : Debug dev : sendcontent free: 177748 chunk size:1360
> 153070 : Debug dev : sendcontent free: 176036 chunk size:1360
> 153080 : Debug dev : sendcontent free: 177748 chunk size:1360
> 153091 : Debug dev : sendcontent free: 176036 chunk size:1360
> 153104 : Debug dev : sendcontent free: 177752 chunk size:1360
> 153117 : Info   :  Webserver 1 Arguments: 0: 'cmd' length: 8
> 153118 : Info   : HTTP: GPIO,2,1
> 153120 : Debug  : HTTP after parseTemplate: GPIO,2,1
> 153123 : Debug  : Command: GPIO
> 153124 : Debug  : GPIO,2,1
> 153124 : Debug  : Par1: 2 Par2: 1 Par3: 0 Par4: 0 Par5: 0
> 153126 : Info   : GPIO : port#2: set to 1
> 153131 : Debug dev : sendcontent free: 178756 chunk size:1360
> 153143 : Debug dev : sendcontent free: 177172 chunk size:1360
> 153157 : Debug dev : sendcontent free: 179284 chunk size:1360
> 153168 : Debug dev : sendcontent free: 174404 chunk size:271
> 153178 : Debug dev : sendcontent free: 175912 chunk size:0
> 162779 : Debug dev : sendcontent free: 177736 chunk size:1360
> 162790 : Debug dev : sendcontent free: 176008 chunk size:1360
> 162801 : Debug dev : sendcontent free: 177732 chunk size:1360
> 162812 : Debug dev : sendcontent free: 176020 chunk size:1360
> 162826 : Debug dev : sendcontent free: 177736 chunk size:1360
> 162842 : Debug dev : sendcontent free: 179228 chunk size:317
> 162864 : Debug dev : sendcontent free: 179164 chunk size:1360
> 162877 : Debug dev : sendcontent free: 177580 chunk size:1360
> 162889 : Debug dev : DEBUG: String size:9837
> 162892 : Debug dev : sendcontent free: 179228 chunk size:1344
> 162902 : Debug dev : sendcontent free: 174132 chunk size:0
> 164092 : Debug dev : sendcontent free: 177300 chunk size:29
> 164102 : Debug dev : sendcontent free: 175880 chunk size:0
> 164126 : Debug  : HTML : Request file /connecttest.txt
> 164897 : Debug dev : sendcontent free: 172464 chunk size:1360
> 164905 : Debug dev : sendcontent free: 171056 chunk size:1360
> 164913 : Debug dev : sendcontent free: 172440 chunk size:1360
> 164925 : Debug dev : sendcontent free: 176008 chunk size:1360
> 164939 : Debug dev : sendcontent free: 177560 chunk size:1360
> 164952 : Info   :  Webserver 2 Arguments: 0: 'index' length: 1 1: 'page' length: 1
> 164958 : Debug dev : sendcontent free: 179120 chunk size:1360
> 164973 : Debug dev : sendcontent free: 177528 chunk size:1360
> 164988 : Debug dev : sendcontent free: 179104 chunk size:1360
> 165004 : Debug dev : sendcontent free: 177512 chunk size:1360
> 165019 : Debug dev : sendcontent free: 179104 chunk size:1360
> 165034 : Debug dev : sendcontent free: 177592 chunk size:1360
> 165052 : Debug dev : DEBUG: String size:14960
> 165054 : Debug dev : sendcontent free: 179092 chunk size:1043
> 165063 : Debug dev : sendcontent free: 173984 chunk size:0
> 168427 : Info   : WD   : Uptime 3 ConnectFailures 0 FreeMem 180556 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 168429 : Debug More : UDP  : Send Sysinfo message
> 169316 : Debug  : LoopStats: shortestLoop: 47 longestLoop: 2088719 avgLoopDuration: 65.22 loopCounterMax: 638297 loopCounterLast: 449428
> 169318 : Debug  : Scheduler stats: (called/tasks/max_length/idle%) 449428/2085/6/93.11
> 173534 : Debug dev : sendcontent free: 177416 chunk size:1360
> 173547 : Debug dev : sendcontent free: 179000 chunk size:1360
> 173557 : Debug dev : sendcontent free: 177416 chunk size:1360
> 173569 : Debug dev : sendcontent free: 174016 chunk size:1360
> 173581 : Debug dev : sendcontent free: 177416 chunk size:1360
> 173595 : Info   :  Webserver 5 Arguments: 0: 'index' length: 1 1: 'page' length: 1 2: 'TDNUM' length: 2 3: 'edit' length: 1 4: 'page' length: 1
> 173598 : Debug dev : Write settings: CustomTaskSettings index: 2
> 175027 : Debug dev : Write settings: TaskSettings index: 2
> 175029 : Info   : SaveToFile: free stack: 7096
> 176684 : Info   : FILE : Saved config.dat offset: 36864 size: 536
> 176685 : Info   : SaveToFile: free stack after: 7096
> 176687 : Info   : SaveToFile: free stack: 7176
> 178890 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 178891 : Info   : SaveToFile: free stack after: 7176
> 178893 : Debug dev : Write settings: ExtendedControllerCredentials index: 0
> 178909 : Info   : SaveToFile: free stack: 6728
> 178983 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 178984 : Info   : SaveToFile: free stack after: 6728
> 178986 : Debug dev : Read settings: TaskSettings index: 2
> 179035 : Debug dev : Read settings: TaskSettings index: 2
> 179062 : Debug dev : sendcontent free: 179052 chunk size:1360
> 179080 : Info   : BMx280: SHOW_ERRORSTATE_OPT
> 179083 : Debug dev : sendcontent free: 177404 chunk size:1360
> 179100 : Debug dev : sendcontent free: 179052 chunk size:1360
> 179116 : Debug dev : DEBUG: String size:10880
> 179121 : Debug dev : sendcontent free: 177324 chunk size:1360
> 179136 : Debug dev : sendcontent free: 175740 chunk size:207
> 179145 : Debug dev : sendcontent free: 173540 chunk size:0
> 179167 : Info   : EVENT: TaskExit#=3,0
> 179209 : Debug  : Rules : Read 0 lines from rules1.txt
> 179232 : Debug  : EVENT: TaskExit#=3,0 Processing time:66 milliSeconds
> 181246 : Info   : EVENT: Clock#Time=Sat,16:41
> 181247 : Debug  : EVENT: Clock#Time=Sat,16:41 Processing time:1 milliSeconds
> 193522 : Debug  : HTML : Request file /connecttest.txt
> 198427 : Info   : WD   : Uptime 3 ConnectFailures 0 FreeMem 181824 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 198429 : Debug More : UDP  : Send Sysinfo message
> 198734 : Debug dev : sendcontent free: 176944 chunk size:1360
> 198746 : Debug dev : sendcontent free: 175216 chunk size:1360
> 198756 : Debug dev : sendcontent free: 176944 chunk size:1360
> 198767 : Debug dev : sendcontent free: 175216 chunk size:1360
> 198783 : Debug dev : sendcontent free: 176892 chunk size:1360
> 198799 : Info   :  Webserver 21 Arguments (First 20): 0: 'index' length: 1 1: 'page' length: 1 2: 'TDNUM' length: 2 3: 'TDN' length: 6 4: 'TDE' length: 2 5: 'i2c_addr' length: 3 6: 'p028_elev' length: 3 7: 'p028_tempoffset' length: 1 8: 'p028_err' length: 1 9: 'TDT' length: 1 10: 'TDVN1' length: 4 11: 'TDF1' length: 0 12: 'TDVD1' length: 1 13: 'TDVN2' length: 3 14: 'TDF2' length: 0 15: 'TDVD2' length: 1 16: 'TDVN3' length: 5 17: 'TDF3' length: 0 18: 'TDVD3' length: 1 19: 'edit' length: 1
> 198806 : Debug dev : Write settings: TaskSettings index: 2
> 198824 : Info   : SaveToFile: free stack: 6792
> 199941 : Info   : FILE : Saved config.dat offset: 36864 size: 536
> 199943 : Info   : SaveToFile: free stack after: 6792
> 199966 : Debug dev : Read settings: TaskSettings index: 2
> 200006 : Debug dev : Write settings: TaskSettings index: 2
> 200009 : Info   : SaveToFile: free stack: 7096
> 201124 : Info   : FILE : Saved config.dat offset: 36864 size: 536
> 201126 : Info   : SaveToFile: free stack after: 7096
> 201127 : Info   : SaveToFile: free stack: 7176
> 202686 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 202688 : Info   : SaveToFile: free stack after: 7176
> 202689 : Debug dev : Write settings: ExtendedControllerCredentials index: 0
> 202709 : Info   : SaveToFile: free stack: 6728
> 202789 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 202790 : Info   : SaveToFile: free stack after: 6728
> 202792 : Debug dev : Read settings: TaskSettings index: 2
> 202845 : Info   : BMx280 : Detected BME280
> 202859 : Debug dev : sendcontent free: 178260 chunk size:1360
> 202873 : Info   : BMx280: SHOW_ERRORSTATE_OPT
> 202876 : Debug dev : sendcontent free: 178108 chunk size:1360
> 202897 : Debug dev : sendcontent free: 178108 chunk size:1360
> 202912 : Debug dev : DEBUG: String size:10880
> 202918 : Debug dev : sendcontent free: 177976 chunk size:1360
> 202929 : Debug dev : sendcontent free: 174604 chunk size:261
> 202933 : Debug dev : sendcontent free: 168976 chunk size:0
> 202949 : Info   : EVENT: TaskExit#BME280=3,0
> 203012 : Debug  : Rules : Read 0 lines from rules1.txt
> 203040 : Debug  : EVENT: TaskExit#BME280=3,0 Processing time:90 milliSeconds
> 203049 : Info   : EVENT: TaskInit#BME280=3,1
> 203050 : Debug  : EVENT: TaskInit#BME280=3,1 Processing time:0 milliSeconds
> 203058 : Debug  : LoopStats: shortestLoop: 47 longestLoop: 5644907 avgLoopDuration: 115.12 loopCounterMax: 638297 loopCounterLast: 289191
> 203059 : Debug  : Scheduler stats: (called/tasks/max_length/idle%) 289189/1688/7/94.94
> 204058 : Info   : BME280: dew point 13.90C
> 204072 : Info   : BME280 : Address: 0x76
> 204076 : Info   : BME280 : Temperature: 28.79
> 204079 : Info   : BME280 : Humidity: 40.09
> 204082 : Info   : BME280 : Barometric Pressure: 1011.25
> 204095 : Info   : EVENT: BME280#Temp=28.79
> 204096 : Debug  : EVENT: BME280#Temp=28.79 Processing time:1 milliSeconds
> 204098 : Info   : EVENT: BME280#Hum=40.09
> 204099 : Debug  : EVENT: BME280#Hum=40.09 Processing time:1 milliSeconds
> 204100 : Info   : EVENT: BME280#Press=1011.25
> 204100 : Debug  : EVENT: BME280#Press=1011.25 Processing time:0 milliSeconds
> 209080 : Debug dev : sendcontent free: 177620 chunk size:1360
> 209092 : Debug dev : sendcontent free: 175896 chunk size:1360
> 209102 : Debug dev : sendcontent free: 177612 chunk size:1360
> 209113 : Debug dev : sendcontent free: 175928 chunk size:1360
> 209131 : Debug dev : sendcontent free: 177568 chunk size:1360
> 209150 : Debug dev : sendcontent free: 179060 chunk size:317
> 209188 : Debug dev : sendcontent free: 179060 chunk size:1360
> 209200 : Debug dev : sendcontent free: 177476 chunk size:1360
> 209215 : Debug dev : sendcontent free: 179060 chunk size:1360
> 209224 : Debug dev : DEBUG: String size:11197
> 209227 : Debug dev : sendcontent free: 177476 chunk size:458
> 209239 : Debug dev : sendcontent free: 175672 chunk size:0
> 210393 : Debug dev : sendcontent free: 177668 chunk size:287
> 210403 : Debug dev : sendcontent free: 175872 chunk size:0
> 210803 : Debug dev : sendcontent free: 177392 chunk size:1360
> 210814 : Debug dev : sendcontent free: 175668 chunk size:1360
> 210825 : Debug dev : sendcontent free: 177384 chunk size:1360
> 210836 : Debug dev : sendcontent free: 175668 chunk size:1360
> 210849 : Debug dev : sendcontent free: 177384 chunk size:1360
> 210862 : Info   :  Webserver 2 Arguments: 0: 'index' length: 1 1: 'page' length: 1
> 210869 : Debug dev : sendcontent free: 178760 chunk size:1360
> 210885 : Debug dev : sendcontent free: 177168 chunk size:1360
> 210900 : Debug dev : sendcontent free: 178744 chunk size:1360
> 210915 : Debug dev : sendcontent free: 177152 chunk size:1360
> 210932 : Debug dev : sendcontent free: 178744 chunk size:1360
> 210947 : Debug dev : sendcontent free: 177232 chunk size:1360
> 210965 : Debug dev : DEBUG: String size:14960
> 210967 : Debug dev : sendcontent free: 178744 chunk size:1043
> 210980 : Debug dev : sendcontent free: 173400 chunk size:0
> 217669 : Debug dev : sendcontent free: 177224 chunk size:1360
> 217682 : Debug dev : sendcontent free: 178808 chunk size:1360
> 217693 : Debug dev : sendcontent free: 177216 chunk size:1360
> 217704 : Debug dev : sendcontent free: 174900 chunk size:1360
> 217719 : Debug dev : sendcontent free: 177224 chunk size:1360
> 217732 : Info   :  Webserver 5 Arguments: 0: 'index' length: 1 1: 'page' length: 1 2: 'TDNUM' length: 2 3: 'edit' length: 1 4: 'page' length: 1
> 217736 : Debug dev : Write settings: CustomTaskSettings index: 3
> 218876 : Debug dev : Write settings: TaskSettings index: 3
> 218878 : Info   : SaveToFile: free stack: 7096
> 220190 : Info   : FILE : Saved config.dat offset: 38912 size: 536
> 220191 : Info   : SaveToFile: free stack after: 7096
> 220193 : Debug dev : Read settings: TaskSettings index: 2
> 220207 : Debug dev : Read settings: TaskSettings index: 3
> 220217 : Info   : SaveToFile: free stack: 7176
> 221745 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 221746 : Info   : SaveToFile: free stack after: 7176
> 221747 : Debug dev : Write settings: ExtendedControllerCredentials index: 0
> 221753 : Info   : SaveToFile: free stack: 6728
> 221813 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 221814 : Info   : SaveToFile: free stack after: 6728
> 221816 : Debug dev : Read settings: TaskSettings index: 3
> 221853 : Debug dev : Read settings: TaskSettings index: 3
> 221871 : Debug dev : sendcontent free: 178928 chunk size:1149
> 221897 : Debug dev : sendcontent free: 178928 chunk size:1360
> 221914 : Debug dev : sendcontent free: 177244 chunk size:1360
> 221933 : Debug dev : sendcontent free: 177108 chunk size:1360
> 221947 : Debug dev : sendcontent free: 177232 chunk size:1360
> 221963 : Debug dev : sendcontent free: 178928 chunk size:1360
> 221976 : Debug dev : sendcontent free: 177252 chunk size:1360
> 221993 : Debug dev : sendcontent free: 178928 chunk size:1360
> 222008 : Debug dev : sendcontent free: 177340 chunk size:1360
> 222023 : Debug dev : DEBUG: String size:18829
> 222027 : Debug dev : sendcontent free: 178784 chunk size:946
> 222038 : Debug dev : sendcontent free: 173876 chunk size:0
> 222057 : Info   : EVENT: TaskExit#=4,0
> 222078 : Debug  : Rules : Read 0 lines from rules1.txt
> 222091 : Debug  : EVENT: TaskExit#=4,0 Processing time:34 milliSeconds
> 227112 : Debug  : HTML : Request file /connecttest.txt
> 228428 : Info   : WD   : Uptime 4 ConnectFailures 0 FreeMem 181564 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 228429 : Debug More : UDP  : Send Sysinfo message
> 229316 : Debug  : LoopStats: shortestLoop: 47 longestLoop: 5644907 avgLoopDuration: 88.96 loopCounterMax: 638297 loopCounterLast: 290224
> 229318 : Debug  : Scheduler stats: (called/tasks/max_length/idle%) 290223/1530/7/72.87
> 241106 : Info   : EVENT: Clock#Time=Sat,16:42
> 241107 : Debug  : EVENT: Clock#Time=Sat,16:42 Processing time:1 milliSeconds
> 249416 : Debug dev : sendcontent free: 176548 chunk size:1360
> 249428 : Debug dev : sendcontent free: 174808 chunk size:1360
> 249440 : Debug dev : sendcontent free: 176536 chunk size:1360
> 249454 : Debug dev : sendcontent free: 178132 chunk size:1360
> 249469 : Debug dev : sendcontent free: 176484 chunk size:1360
> 249485 : Info   :  Webserver 28 Arguments (First 20): 0: 'index' length: 1 1: 'page' length: 1 2: 'TDNUM' length: 2 3: 'TDN' length: 7 4: 'serPort' length: 1 5: 'i2cuart_addr' length: 2 6: 'i2cuart_ch' length: 1 7: 'taskdevicepin1' length: 2 8: 'taskdevicepin2' length: 1 9: 'rstpin' length: 2 10: 'pwrpin' length: 2 11: 'p053_model' length: 1 12: 'p053_output' length: 1 13: 'p053_events' length: 1 14: 'p053_delay_wake' length: 2 15: 'p053_oversample' length: 2 16: 'TDT' length: 2 17: 'TDVN1' length: 5 18: 'TDF1' length: 0 19: 'TDVD1' length: 1
> 249492 : Debug dev : Write settings: TaskSettings index: 3
> 249502 : Info   : SaveToFile: free stack: 6792
> 250656 : Info   : FILE : Saved config.dat offset: 38912 size: 536
> 250658 : Info   : SaveToFile: free stack after: 6792
> 250659 : Debug dev : Read settings: TaskSettings index: 2
> 250683 : Debug dev : Read settings: TaskSettings index: 3
> 250714 : Debug dev : Write settings: TaskSettings index: 3
> 250717 : Info   : SaveToFile: free stack: 7096
> 251836 : Info   : FILE : Saved config.dat offset: 38912 size: 536
> 251838 : Info   : SaveToFile: free stack after: 7096
> 251839 : Debug dev : Read settings: TaskSettings index: 2
> 251861 : Debug dev : Read settings: TaskSettings index: 3
> 251878 : Info   : SaveToFile: free stack: 7176
> 253434 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 253436 : Info   : SaveToFile: free stack after: 7176
> 253437 : Debug dev : Write settings: ExtendedControllerCredentials index: 0
> 253452 : Info   : SaveToFile: free stack: 6728
> 253526 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 253527 : Info   : SaveToFile: free stack after: 6728
> 253529 : Debug dev : Read settings: TaskSettings index: 3
> 253574 : Debug dev : Read settings: TaskSettings index: 3
> 253597 : Debug dev : sendcontent free: 177828 chunk size:1156
> 253623 : Debug dev : sendcontent free: 177828 chunk size:1360
> 253640 : Debug dev : sendcontent free: 176164 chunk size:1360
> 253658 : Debug dev : sendcontent free: 177728 chunk size:1360
> 253673 : Debug dev : sendcontent free: 176148 chunk size:1360
> 253684 : Debug dev : sendcontent free: 171492 chunk size:1360
> 253699 : Debug dev : sendcontent free: 176212 chunk size:1360
> 253715 : Debug dev : sendcontent free: 177828 chunk size:1360
> 253731 : Debug dev : sendcontent free: 176244 chunk size:1360
> 253747 : Debug dev : DEBUG: String size:18836
> 253752 : Debug dev : sendcontent free: 177692 chunk size:957
> 253762 : Debug dev : sendcontent free: 172604 chunk size:0
> 253779 : Info   : EVENT: TaskExit#PMS7003=4,0
> 253817 : Debug  : Rules : Read 0 lines from rules1.txt
> 253835 : Debug  : EVENT: TaskExit#PMS7003=4,0 Processing time:56 milliSeconds
> 253848 : Info   : EVENT: TaskExit#PMS7003=4,0
> 253849 : Debug  : EVENT: TaskExit#PMS7003=4,0 Processing time:2 milliSeconds
> 258427 : Info   : WD   : Uptime 4 ConnectFailures 0 FreeMem 179376 WiFiStatus WL_CONNECTED 3 ESPeasy internal wifi status: Conn. IP Init
> 258429 : Debug More : UDP  : Send Sysinfo message
> 258851 : Debug  : HTML : Request file /connecttest.txt
> 259316 : Debug  : LoopStats: shortestLoop: 47 longestLoop: 5644907 avgLoopDuration: 100.32 loopCounterMax: 638297 loopCounterLast: 294561
> 259318 : Debug  : Scheduler stats: (called/tasks/max_length/idle%) 294560/1815/7/89.09
> 260991 : Debug dev : sendcontent free: 176636 chunk size:1360
> 261003 : Debug dev : sendcontent free: 174920 chunk size:1360
> 261014 : Debug dev : sendcontent free: 176644 chunk size:1360
> 261025 : Debug dev : sendcontent free: 174916 chunk size:1360
> 261040 : Debug dev : sendcontent free: 176580 chunk size:1360
> 261057 : Info   :  Webserver 29 Arguments (First 20): 0: 'index' length: 1 1: 'page' length: 1 2: 'TDNUM' length: 2 3: 'TDN' length: 7 4: 'TDE' length: 2 5: 'serPort' length: 1 6: 'i2cuart_addr' length: 2 7: 'i2cuart_ch' length: 1 8: 'taskdevicepin1' length: 2 9: 'taskdevicepin2' length: 1 10: 'rstpin' length: 2 11: 'pwrpin' length: 2 12: 'p053_model' length: 1 13: 'p053_output' length: 1 14: 'p053_events' length: 1 15: 'p053_delay_wake' length: 2 16: 'p053_oversample' length: 2 17: 'TDT' length: 2 18: 'TDVN1' length: 5 19: 'TDF1' length: 0
> 261064 : Debug dev : Write settings: TaskSettings index: 3
> 261077 : Info   : SaveToFile: free stack: 6792
> 262215 : Info   : FILE : Saved config.dat offset: 38912 size: 536
> 262217 : Info   : SaveToFile: free stack after: 6792
> 262218 : Debug dev : Read settings: TaskSettings index: 2
> 262251 : Debug dev : Read settings: TaskSettings index: 3
> 262287 : Debug dev : Write settings: TaskSettings index: 3
> 262290 : Info   : SaveToFile: free stack: 7096
> 263433 : Info   : FILE : Saved config.dat offset: 38912 size: 536
> 263435 : Info   : SaveToFile: free stack after: 7096
> 263436 : Debug dev : Read settings: TaskSettings index: 2
> 263469 : Debug dev : Read settings: TaskSettings index: 3
> 263491 : Info   : SaveToFile: free stack: 7176
> 265072 : Info   : FILE : Saved config.dat offset: 0 size: 3004
> 265073 : Info   : SaveToFile: free stack after: 7176
> 265075 : Debug dev : Write settings: ExtendedControllerCredentials index: 0
> 265093 : Info   : SaveToFile: free stack: 6728
> 265171 : Info   : FILE : Saved security.dat offset: 1024 size: 6
> 265172 : Info   : SaveToFile: free stack after: 6728
> 265174 : Debug dev : Read settings: TaskSettings index: 3
> 265214 : Debug  : PMSx003 : config 13 4 -1 -1
> 265215 : Info   : PMSx003 : using hardware serialGuru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
> 
> Core  1 register dump:
> PC      : 0x40093c1a  PS      : 0x00060330  A0      : 0x8009426e  A1      : 0x3ffb2180
> A2      : 0xffffffff  A3      : 0x00000017  A4      : 0x3ffc7bbc  A5      : 0x0000abab
> A6      : 0x00060323  A7      : 0x00060320  A8      : 0x00000002  A9      : 0xffffffff
> A10     : 0x00000002  A11     : 0x00000110  A12     : 0x00000017  A13     : 0x0000abab
> A14     : 0x00060323  A15     : 0x00060323  SAR     : 0x00000003  EXCCAUSE: 0x0000001c
> EXCVADDR: 0x0000001b  LBEG    : 0x4008c43d  LEND    : 0x4008c44d  LCOUNT  : 0xffffffff
> 
> 
> Backtrace:0x40093c17:0x3ffb21800x4009426b:0x3ffb21a0 0x400944d8:0x3ffb21c0 0x4008499a:0x3ffb21e0 0x400849ad:0x3ffb2200 0x40084bc8:0x3ffb2220 0x401f2af1:0x3ffb2240 0x401ec9be:0x3ffb2280 0x401e8b0e:0x3ffb22e0 0x400e2786:0x3ffb2320 0x401bd933:0x3ffb2350 0x401384f7:0x3ffb23b0 0x40194784:0x3ffb24c0 0x401e45ff:0x3ffb25b0 0x402758d5:0x3ffb2660 0x400d80a6:0x3ffb2680 0x400dcc7d:0x3ffb26a0 0x400dcd2d:0x3ffb26e0 0x400dce35:0x3ffb2740 0x4018e0ce:0x3ffb2790 0x4018e385:0x3ffb27c0 0x401265f3:0x3ffb2800 0x401eccd8:0x3ffb2820
> 
> 
> 
> 
> ELF file SHA256: 0000000000000000
> 
> Rebooting...
> ets Jun  8 2016 00:22:57
> 
> rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
> configsip: 0, SPIWP:0xee
> clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
> mode:DOUT, clock div:2
> load:0x3fff0030,len:184
> load:0x40078000,len:12616
> load:0x40080400,len:2916
> entry 0x400805c4
> ▒U47 : Info   :
> 
> INIT : Booting version: ESP_Easy_mega_20220813_max_ESP32_16M8M_LittleFS, (Self built) feature/several-small-improvements-2022-08_5bd455d (ESP32 SDK v4.4.3)
> 49 : Info   : INIT : Free RAM:266608
> 50 : Info   : INIT : Soft Reboot #2 Last Action before Reboot: Const Interval: TIMER_20MSEC Last systime: 1660387340 - Restart Reason: CPU0: Software reset CPU CPU1: Software reset CPU
> 51 : Info   : FS   : Mounting...
> 105 : Info   : FS   : Mount successful, used 151552 bytes of 8318976
> 204 : Info   : CRC  : SecuritySettings CRC   ...OK
> 230 : Info   : INIT : I2C
> 233 : Info   : INIT : SPI Init (without CS)
> 235 : Info   : ExtRTC: Read external time source: 1660387346
> 236 : Info   : Time set to 1660387346.000
> 237 : Info   : Current Time Zone: STD time start: 2022-10-30 03:00:00 offset: 360 min
> 238 : Info   : Local time: 2022-08-13 16:42:26
> 240 : Info   : WiFi : Start network scan all channels
> 343 : Info   : WiFi : STA Started
> 6956 : Info   : WiFi : Scan finished, found: 11
> 6960 : Info   : WiFi : Best AP candidate: access_point8 88:DC:96:26:7F:0A Ch:9 (-58dBm) WPA2/PSK
> 6961 : Info   : WIFI : Connecting access_point8 88:DC:96:26:7F:0A Ch:9 (-58dBm) WPA2/PSK attempt #0
> 6971 : Info   : WIFI : Arduino wifi status: WL_DISCONNECTED 6 ESPeasy internal wifi status: DISCONNECTED
> 6995 : Info   : INIT : Free RAM:219256
> 7048 : Info   : PMSx003 : using hardware serialGuru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
> 
> Core  1 register dump:
> PC      : 0x40093c1a  PS      : 0x00060f30  A0      : 0x8009426e  A1      : 0x3ffb22e0
> A2      : 0xffffffff  A3      : 0x00000017  A4      : 0x3ffc7bbc  A5      : 0x0000abab
> A6      : 0x00060f23  A7      : 0x00060f20  A8      : 0x00000002  A9      : 0xffffffff
> A10     : 0x00000002  A11     : 0x00000110  A12     : 0x00000017  A13     : 0x0000abab
> A14     : 0x00060f23  A15     : 0x00060f23  SAR     : 0x00000003  EXCCAUSE: 0x0000001c
> EXCVADDR: 0x0000001b  LBEG    : 0x4008c43d  LEND    : 0x4008c44d  LCOUNT  : 0xffffffff
> 
> 
> Backtrace:0x40093c17:0x3ffb22e00x4009426b:0x3ffb2300 0x400944d8:0x3ffb2320 0x4008499a:0x3ffb2340 0x400849ad:0x3ffb2360 0x40084bc8:0x3ffb2380 0x401f2af1:0x3ffb23a0 0x401ec9be:0x3ffb23e0 0x401e8b0e:0x3ffb2440 0x400e2786:0x3ffb2480 0x401bd933:0x3ffb24b0 0x401384f7:0x3ffb2510 0x4019426f:0x3ffb2620 0x40194586:0x3ffb2670 0x401a5872:0x3ffb2760 0x4018e869:0x3ffb27a0 0x401265eb:0x3ffb2800 0x401eccc6:0x3ffb2820
> 
> 
> 
> 
> ELF file SHA256: 0000000000000000
> 
> Rebooting...
>

--- a/tools/pio/post_esp32.py
+++ b/tools/pio/post_esp32.py
@@ -40,7 +40,11 @@ def esp32_create_combined_bin(source, target, env):
     flash_size = env.BoardConfig().get("upload.flash_size")
     flash_freq = env.BoardConfig().get("build.f_flash", '40m')
     flash_freq = flash_freq.replace('000000L', 'm')
-    flash_mode = env.BoardConfig().get("build.flash_mode")
+    flash_mode = env.BoardConfig().get("build.flash_mode", "dout")
+    if flash_mode == "qio":
+        flash_mode = "dio"
+    elif flash_mode == "qout":
+        flash_mode = "dout"
     cmd = [
         "--chip",
         chip,

